SkietbaanBE/SkietbaanBE/Controllers/CompetitionController.cs

```diff
@@ -37,7 +37,16 @@ public IEnumerable<Competition> GetCompetitions()
         public IEnumerable<Competition> GetAllCompetitions()
        {
             //get the competitions where(Status == true / false)
-             return _context.Competitions.ToArray<Competition>().OrderBy(x=> (x.Status == false)).ThenBy(x =>(x.Name));
+            List<Competition> CompList = (from C in _context.Competitions
```

Comment:
try naming variables in fill names

Reply:
Like instead of CompList or CompetitionList?

Reply:
yes